OpenQA/Qemu/Proc.pm

```diff
@@ -163,7 +163,7 @@ sub configure_blockdevs {
             $drive = $bdc->add_new_drive($node_id, $hdd_model, $size);
         }
 
-        if ($i == 1 && $uefi && $bootfrom eq 'disk') {
+        if ($i == 1 && $bootfrom eq 'disk') {
```

Comment:
We don't care about UEFI anymore? Hmm?

Reply:
It no longer requires UEFI, but will be still used with it, unless I have got the logic wrong?

Reply:
Doesn't look like it's used elsewhere, at least within this method

Reply:
ah, removed.

Reply:
:)